modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/VirtualFlowTest.java

```diff
@@ -635,7 +655,8 @@ private void assertMatch(List<IndexedCell> control, AbstractList<IndexedCell> li
      * Tests the size and position of all the cells to make sure they were
      * laid out properly.
      */
-    @Test public void testCellLayout_CellSizes_AfterLayout() {
+    @Test
```

Comment:
Only new line is added after @Test tag. There are a lot of changes of this type which I presume were done by the IDE. I think, we can revert these safely as not to cause unnecessary diffs.

Reply:
reverted